script.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var citySelected = document.getElementById("city-select");
+
+document.getElementById("city-select").onchange = function() {

```

Comment:
(修正不要)
es6からは、フレームワークの事情を除けばfunctionキーワードを廃止してアロー関数に置き換えるとよいそうです。

```suggestion
document.getElementById("city-select").onchange = () => {
```